content/en/cloud_cost_management/_index.md

```diff
@@ -165,21 +165,75 @@ The following out-of-the-box tags are also available for filtering and grouping
 
 ### Rule types
 
+<div class="alert alert-info"><strong>Note</strong>: A maximum of 100 rules can be created. </div>
+
 **Create new tag** - This allows you to create a new tag (key + value) based on the presence of existing tags. For example, you can create a rule to tag all resources that are part of team A, B, or C, and also run a specified application, with a new `cost-center:webstore` tag.
 
-**Alias existing tag keys** - This allows you to use values from an existing tag, to map to a more standardized tag key. For example, if you’re looking to standardize across your organization to use the `application` tag key, but several teams have a variation of that tag (like `app`, `web-app`, or `apps`), you can alias `apps` to `application`. Each alias tag rule allows you to alias a maximum of 25 tag keys to a new tag.   
+**Alias existing tag keys** - This allows you to use values from an existing tag, to map to a more standardized tag key. For example, if you're looking to standardize across your organization to use the `application` tag key, but several teams have a variation of that tag (like `app`, `web-app`, or `apps`), you can alias `apps` to `application`. Each alias tag rule allows you to alias a maximum of 25 tag keys to a new tag.   
 
 The rule stops executing for each resource, once a first match is found. For example, if a resource already has a `web-app` tag, then the rule no longer attempts to identify an `apps` or `service` tag. 
 
-**Note:** A maximum of 100 rules can be created. 
+[1]: https://docs.aws.amazon.com/cur/latest/userguide/cur-create.html
+[2]: https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/consolidated-billing.html
+[3]: https://us-east-1.console.aws.amazon.com/cost-management/home?region=us-east-1#/settings
+[4]: https://docs.aws.amazon.com/cur/latest/userguide/view-cur.html
+[5]: https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html
+[6]: https://docs.aws.amazon.com/cur/latest/userguide/data-dictionary.html
+{{% /tab %}}
+
+{{% tab "Azure" %}}
+<div class="alert alert-warning">Azure Cloud Cost Management is in private beta. Fill out this <a href="https://docs.google.com/forms/d/e/1FAIpQLSftAIq_g4GxBAKdWV5OjP0Ui4CAjWTzH3YCKy3n930gMz0Krg/viewform?usp=sf_link">form</a> to request access.</div>
+
+To use Azure Cloud Cost Management in Datadog, you must have an Azure account and have the following billing exports set up: **amortized** and **actual exports**. Additionally, Datadog must have permissions to read the exports from the container.
+
+**Note**: If you use the recommended [Datadog Resource method][1] through the Azure portal to set up Datadog, App Registration creation is still required to support Cloud Cost Management.
+
+### Schedule exports
+
+1. Navigate to [Exports][2] under Azure portal's *Cost Management + Billing*.
```

Comment:
Opening this link took me to Exports under Cost Management, but Billing seems to be separate:
https://a.cl.ly/Jruyd4rj

Could you confirm that it should say `+ Billing` here?

Reply:
Thanks for checking this! PM confirmed that the link should be changed, but the portal heading should still be Cost Management + Billing.